test/resources/code/python/feature_store_py_spark/py_spark_feature_store_ingestion.py

```diff
@@ -0,0 +1,82 @@
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Can you add a test for `CreateProcessingJob` to make sure Processing Job doesn't get impacted?

Reply:
The processing job is created via the `PySparkProcessor` which will create processing job with the image to be released and run. To be more specific, this `py_spark_feature_store_ingestion.py` will be executed by `test_sagemaker_feature_store_ingestion_multinode`